src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -352,6 +355,7 @@ export class SettingsEditor2 extends EditorPane {
 	}
 
 	override async setInput(input: SettingsEditor2Input, options: ISettingsEditorOptions | undefined, context: IEditorOpenContext, token: CancellationToken): Promise<void> {
+		this.inputChangeListener.clear();
```

Comment:
Do we need the clear in this case? Setting a new value automatically clears out the old one.

Reply:
Good catch, it seems not needed!